zerver/views/registration.py

```diff
@@ -877,6 +881,56 @@ def redirect_to_email_login_url(email: str) -> HttpResponseRedirect:
     return HttpResponseRedirect(redirect_url)
 
 
+def realm_import_post_process(
+    request: HttpRequest,
+    confirmation_key: str,
+) -> HttpResponse:
+    try:
+        # We don't use get_object_from_key because we're fine if it's been marked used
+        confirmation = Confirmation.objects.get(
+            confirmation_key=confirmation_key, type__in=[Confirmation.REALM_CREATION]
+        )
+    except Confirmation.DoesNotExist:
+        raise JsonableError(_("Unauthenticated"))
```

Comment:
Probably rather than doing this and adding an exclude to the linter, we should add a `allow_used` optional parameter to `get_object_from_key`.

Also, is JsonableError the right error format here? I would think this is a main HTML page, not a thing we visit with JS, and thus should serve an error page.

Reply:
Done.

> Also, is JsonableError the right error format here?
No, changed it render a confirmation key error page.